ui-charts/src/spaceTimeChart/components/PathLayer.tsx

```diff
@@ -66,6 +67,13 @@ export type PathLayerProps = {
   color: string;
   pickingTolerance?: number;
   level?: PathLevel;
+  border?: {
+    offset: number;
+    level: PathLevel;
```

Comment:
Seems like this is unused

Reply:
thanks, it was a leftover from a previous approach